fix formatting and wording (#269)
AdguardTeam/KnowledgeBaseDNS: docs/private-dns/connect-devices/routers/xiaomi.md

sidebar_position: 11
---

> Xiaomi routers have many advantages: a stable, strong signal, network security, stable operation, and intelligent management. The user can also connect up to 64 devices to a local Wi-Fi network.

Reviewer: два раза stable + не очень понятна идея с at the same time и also
```suggestion
Xiaomi routers have many advantages: a stable, strong signal, network security, robust performance, and smart management. Users can connect up to 64 devices to a local Wi-Fi network.
```

Author: про stable ты права, а про also — ну было совсем плохо, at the same time зачем-то обособленно и слишком много запятых
